README.md

```diff
@@ -39,11 +39,11 @@ Install `python3`, if not installed (in **Ubuntu 20.04**):
 sudo apt install python3.8
 ```
 
-Clone the repo and install the library:
+ Install the library via pip:
 
 ```bash
 
-pip install "urdfModifiers @ git+https://github.com/icub-tech-iit/urdf-modifiers"
+pip install urdfModifiers 
```

Comment:
```suggestion
pip install urdf-modifiers 
```

Reply:
I would also change https://github.com/CarlottaSartore/urdf-modifiers/blob/adding_pypi_ci/setup.cfg#L2 .

Reply:
Argh, but the python module name is `urdfModifiers` .

Reply:
To clarify why I do not like `urdfModifiers`, camel case is not suggested in PEP8 guidelines, see https://peps.python.org/pep-0008/#package-and-module-names, and not so common at all (see https://pypistats.org/top). Anyhow, I don't want to create more friction than necessary, so if you want to go forward with `urdfModifiers`, feel free to proceed.

Reply:
Thinking more about this, I think we should at least call the package name `urdf-modifiers` . We can always change the module name later from `urdfModifiers` to `urdf-modifiers`  in a backward compatible way, but changing python package name would be complex.

Reply:
Done in 71f49ee9c18b9b74a9afd9673cd29e06eca83d60 and  8747f1591c7b9f3425dca7cffcdbf5c985122d5e